demo/admin/src/common/blocks/MediaBlock.tsx

```diff
@@ -8,5 +10,27 @@ export const MediaBlock: BlockInterface = createOneOfBlock({
     displayName: <FormattedMessage id="mediaBlock.displayName" defaultMessage="Media" />,
     allowEmpty: false,
     variant: "toggle",
+    labels: {
+        image: (
+            <Tooltip trigger="hover" title={<FormattedMessage id="pages.blocks.media.image" defaultMessage="Image" />}>
```

Comment:
We can reuse the display name of the blocks here, e.g. `title={DamImageBlock.displayName}`